tests/testTriggers.go

```diff
@@ -4548,6 +4552,92 @@ func TriggerPoolAddDisk(contexts *[]*scheduler.Context, recordChan *chan *EventR
 	updateMetrics(*event)
 }
 
+// TriggerPoolAutoDisk performs auto-disk on the storage pools for the given contexts
+func TriggerPoolAutoDisk(contexts *[]*scheduler.Context, recordChan *chan *EventRecord) {
+	defer ginkgo.GinkgoRecover()
+	defer endLongevityTest()
+	startLongevityTest(PoolAutoDisk)
+	event := &EventRecord{
+		Event: Event{
+			ID:   GenerateUUID(),
+			Type: PoolAutoDisk,
+		},
+		Start:   time.Now().Format(time.RFC1123),
+		Outcome: []error{},
+	}
+	defer func() {
+		event.End = time.Now().Format(time.RFC1123)
+		*recordChan <- event
+	}()
+	setMetrics(*event)
+	chaosLevel := getPoolExpandPercentage(PoolAutoDisk)
+	stepLog := fmt.Sprintf("get storage pools and perform auto-disk by %v percentage on it ", chaosLevel)
+	Step(stepLog, func() {
+		log.InfoD(stepLog)
+		poolsToBeResized, err := getStoragePoolsToExpand()
+		if err != nil {
+			log.Error(err.Error())
+			UpdateOutcome(event, err)
+		}
+		log.InfoD("Pools to auto-disk [%v]", poolsToBeResized)
+		var wg sync.WaitGroup
+		for _, pool := range poolsToBeResized {
+			//Skipping pool resize if pool rebalance is enabled for the pool
+			if !isPoolRebalanceEnabled(pool.Uuid) {
+				//Initiating multiple pool expansions by auto-disk
```

Comment:
please change it to auto

Reply:
Done. Changed auto-disk to auto everywhere.

Reply:
Added Expansion prefix.